torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -1970,6 +1970,17 @@ Tensor elu_double_backward(
     }
 }
 
+Tensor sinc_backward(const Tensor& grad, const Tensor& self) {
+  Tensor self_scaled = at::ones_like(self);
+  Tensor scale_v;
+  Tensor pi = wrapped_scalar_tensor(Scalar(M_PI));
+  scale_v = grad *
+      ((pi * self * (pi * self).cos() - (pi * self).sin()) / (pi * self * self))
+          .conj();
+  scale_v.masked_fill_(self == 0, 0);
```

Comment:
From the failing test, looks like complex float/double isn't supported for masked_fill on CUDA. If we really want to unblock this, we'll have to drop support for complex on CUDA until that is supported for masked_fill, and we'll need to update the OpInfo to no longer test complex on CUDA. This is bc breaking though cc @gchanan

Reply:
https://github.com/pytorch/pytorch/blob/aa8714dfedc73c67524e2394fe04d115f0783a09/torch/testing/_internal/common_methods_invocations.py

Reply:
You can try to use where here that supports complex dtypes afaik
The python version would be this:
```
torch.where(mask, a, torch.tensor(0, dtype=a.dtype, device=a.device))
```